templates/server.yaml.j2

```diff
@@ -97,6 +101,8 @@ spec:
             - name: glusterfsd-datadir
               mountPath: "/bricks"
               mountPropagation: Bidirectional
+            - name: glusterfsd-volfilesdir
+              mountPath: "/var/lib/gluster"
```

Comment:
What is this? Why is this required?

Reply:
I saw a log from quotad container, let me recheck. May be a setup issue. If it works without this change I will remove.